tracing-subscriber/src/fmt/fmt_layer.rs

```diff
@@ -492,6 +492,22 @@ where
         }
     }
 
+    /// Sets whether or not the length of the [name] of the current thread is
+    /// tracked for padding.
+    ///
+    /// [name]: std::thread#naming-threads
```

Comment:
It's probably worth having this method's documentation reference the `with_thread_names` method. Maybe add something like this:
```suggestion
    /// By default, thread names will be padded.
    ///
    /// This setting only has an effect if
    /// [`with_thread_names`](Self::with_thread_names) is `true`.
```

Reply:
Also, let's add something to the docs for `with_thread_names` that refers to this method, like:
> To control whether or not the displayed thread names are padded with extra whitespace for alignment, see the `with_thread_name_padding` method.